fixtures/mark/multiprofile-zoom-led-w3.json

```
@@ -0,0 +1,94 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Multiprofile Zoom Led W3",
+  "categories": ["Dimmer"],
+  "meta": {
+    "authors": ["Ukr"],
+    "createDate": "2024-11-30",
+    "lastModifyDate": "2024-11-30"
+  },
+  "links": {
+    "manual": [
+      "https://www.eafg.es/productos/Docs/Manuales/MULTIPROFILE_ZOOM_LED_W3_17-50_v1_0.pdf"
+    ],
+    "productPage": [
+      "https://products.equipson.es/store-multiprofile-zoom-led-w3-17-50"
+    ],
+    "video": [
+      "https://products.equipson.es/store-multiprofile-zoom-led-w3-17-50"
+    ]
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Strobe": {
+      "capability": {
+        "type": "ShutterStrobe",
+        "shutterEffect": "Open"
+      }
```

```suggestion
      "capability": {
        "type": "ShutterStrobe",
        "shutterEffect": "Strobe",
        "speedStart": "slow",
        "speedEnd": "fast",
        "helpWanted": "At what values is strobe disabled?"
      }
```

Maybe 0-20?  That's the range where Pulse is disabled.